pepper-apis/config/sqlproxy.env.ctmpl

```diff
@@ -1,6 +1,11 @@
-{{$environment := env "ENVIRONMENT"}}
-{{$version := env "VERSION"}}
-{{$response := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
-{{$conf := $response.Data}}
+{{with $environment := env "ENVIRONMENT"}}
+{{with $version := env "VERSION"}}
+{{with $response := secret (printf "secret/pepper/%s/%s/conf" $environment $version)}}
+{{with $conf := $response.Data}}
 
 CLOUDSQL_CONNECTION_LIST={{$conf.googleProjectId}}:us-central1:{{$conf.dbInstanceId}}=tcp:0.0.0.0:3307
+
+{{end}}
+{{end}}
+{{end}}
+{{end}}
```

Comment:
The one template that I had to do more work on then changing `vault` to `secret`.
Had to use `with` and `end` for new ` consul-template` to accept it. Every other template was already good.

Reply:
A fun trick with these now that we're using a newer version- you can use `{{- `, and ` -}}` to suppress leading and trailing newline generation.

Reply:
Ha! Need trick.
Even better: a lot of our templates start with {{with foo=bunchofstuff}}
You can add that - character at the end to not use up whitespace.
(I find the leading blank lines even more annoying than the trailing ones!).
I would like to fix a bunch of templates, but I am going to restrain myself and do that in a separate PR, if you don't mind.

Reply:
That's reasonable- there's a lot here already!